src/igvfd/search_views.py

```diff
@@ -96,14 +99,15 @@ def search_generator(request):
             BasicSearchResponseField(
                 default_item_types=DEFAULT_ITEM_TYPES,
                 reserved_keys=RESERVED_KEYS,
-            )
+            ),
         ]
     )
     return fgr.render()
 
 
 @view_config(route_name='report', request_method='GET', permission='search')
 def report(context, request):
+    validate_report(request)
```

Comment:
I would like to not block this view with type validation if possible. It's basically a search view which doesn't care. Can we just validate in the view that actually generates the TSV?

Reply:
@gabdank  wants the logic implemented at both report view and report download view so the frontend won’t need to display the report view for types report that we don't support for downloading.

Reply:
I think the same logic should be applicable for both: report-view and report-download. 
From talking to Forrest it sounds that the front-end at this point relies on the backend to supply the right answer - for both and I am not sure we should change it.